portfolio/src/main/webapp/style.css

```diff
@@ -1,25 +1,61 @@
 #content {
```

Comment:
Please kindly alphabetize your declarations.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

It would be great to take a look at the google style guide and make sure your code complies.

Reply:
I must have missed that when I was looking over the document. Thank you for catching that!